addons/finger/functions/fnc_keyPress.sqf

```diff
@@ -0,0 +1,55 @@
+/*
+ * Author: TheDrill, PabstMirror
+ * On keypress, point and send position to nearby players
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * Key Handeled <BOOL>
+ *
+ * Example:
+ * [] call ace_finger_fnc_keyPress;
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+private["_fingerPosPrecise", "_playerEyePos", "_sendFingerToPlayers"];
+
+if (!alive ACE_player) exitWith {false};
+// Conditions: canInteract
+if !([ACE_player, ACE_player, ["isNotInside"]] call EFUNC(common,canInteractWith)) exitWith {false};
+//make sure player is dismounted or in a static weapon:
+if ((ACE_player != vehicle ACE_player) && {!((vehicle ACE_player) isKindOf "StaticWeapon")}) exitWith {false};
+//Check camera view (not in GUNNER)
+if !(cameraView in ["INTERNAL", "EXTERNAL"]) exitWith {false};
+//Exit if run recently (run every 1 seconds)
+if (ACE_diagTime < (GVAR(lastFPTime) + FP_ACTION_TIMEOUT)) exitWith {false};
+
+GVAR(lastFPTime) = ACE_diagTime;
+
+_fingerPosPrecise = positionCameraToWorld [0, 0, FP_DISTANCE];
```

Comment:
It would probably make more sense to do a recursive lineintersect search and determine the actual distance to the pointed object. That would allow pointing to nearby objects and positions too, like: "pickup that dropped magazine", "setup the static MG there", "the patient is over there", etc

The 1000m distance could be kept if no object is detected (e.g. while pointing at the horizon).